src/services/Content/ContentQueries.ts

```diff
@@ -20,10 +20,15 @@ export const CONTENT_LIST_KEY = 'CONTENT_LIST_KEY';
 export const REPOSITORY_PARAMS_KEY = 'REPOSITORY_PARAMS_KEY';
 export const CREATE_PARAMS_KEY = 'CREATE_PARAMS_KEY';
 
-export const useContentListQuery = (page: number, limit: number, filterData: FilterData) =>
+export const useContentListQuery = (
+  page: number,
+  limit: number,
+  filterData: FilterData,
+  sortBy: string,
+) =>
   useQuery<ContentListResponse>(
-    [CONTENT_LIST_KEY, page, limit, ...Object.values(filterData)],
-    () => getContentList(page, limit, filterData),
+    [CONTENT_LIST_KEY, page, limit, sortBy, ...Object.values(filterData)],
+    () => getContentList(page, limit, filterData, sortBy),
```

Comment:
You need to add "sortBy" to the delete function to keep our optimistic updates happening. If you slow down the network tab you'll see what I mean after deleting an item.

Reply:
i didn't actually see much of a problem when running on slow 3g, but i went ahead and updated what i think you mean.  let me know!